app/experimenter/experiments/api/v5/query.py

```diff
@@ -1,31 +1,34 @@
 import graphene
 
 from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.constants.nimbus import NimbusConstants
 from experimenter.experiments.models.nimbus import NimbusExperiment
 
 
 class Query(graphene.ObjectType):
-    experiments_by_status = graphene.Field(
-        graphene.List(NimbusExperimentType),
-        status=graphene.Argument(
-            NimbusExperimentType._meta.fields["status"]._type, required=False
-        ),
-    )
-    all_experiments = graphene.List(
+    experiments = graphene.List(
         NimbusExperimentType,
+        description="List Nimbus Experiments.",
+        offset=graphene.Int(),
+        limit=graphene.Int(),
+        status=graphene.Enum(
+            "NimbusExperimentOptionalStatus", NimbusConstants.Status.choices
+        )(),
     )
     experiment_by_slug = graphene.Field(
-        NimbusExperimentType, slug=graphene.String(required=True)
+        NimbusExperimentType,
+        description="Retrieve a Nimbus experiment by its slug.",
+        slug=graphene.String(required=True),
     )
 
-    def resolve_all_experiments(root, info):
-        return NimbusExperiment.objects.all()
+    def resolve_experiments(root, info, offset=0, limit=20, status=None):
+        q = NimbusExperiment.objects
+        if status:
+            q = q.filter(status=status)
+        return q.all()[offset:limit]
```

Comment:
Oh oh, `objects` is a `ModelManager` instance, not a `QuerySet` instance, they have some of the same methods but not all, so the way this is written technically works but could break if we try to do some other operations that querysets support that managers don't.  This should probably be written:

```py
        q = NimbusExperiment.objects.all()
        if status:
            q = q.filter(status=status)
        return q[offset:limit]
```

Reply:
Ahhhh